CodeStyleExercise/ColorsViewController.swift

```diff
@@ -67,28 +64,16 @@ class ColorsViewController : UIViewController, UICollectionViewDataSource, UITex
         }
     }
 
-
-
     // MARK: - IBActions
 
-    // This IBAction is called when the user taps on the background view
-    // It makes the colorsCountTextField resign first responder and sets colors count to the int value of the colorsCountTextField text
     @IBAction func userDidTapOnTheBackgroundView(_ sender: UITapGestureRecognizer) {
```

Comment:
Prefer `handle<target>Tap` or `<blank>ButtonPressed' instead of `userDidX` when naming gesture handlers and button actions.